src/skmatter/metrics/_pairwise.py

```diff
@@ -0,0 +1,230 @@
+from typing import Union
+
+import numpy as np
+from sklearn.metrics.pairwise import (
+    _euclidean_distances,
+    check_array,
+    check_pairwise_arrays,
+)
+
+
+def pairwise_euclidean_distances(
+    X, Y=None, *, Y_norm_squared=None, squared=False, X_norm_squared=None, cell=None
```

Comment:
Since cell is not really a cell but since we currently only support cubic cells and this is length of the cubic cell I would make this also clear in name of the parameter.

```suggestion
    X, Y=None, *, Y_norm_squared=None, squared=False, X_norm_squared=None, cell_length=None
```